### PR TITLE
lib: src: simulations: Added Pi Approximation Simulation #54

### LINKED ISSUE #54: lib: src: simulations: Implement Pi Approximation Simulation
`id: 6`
`Category: Mathematics`

### PR DESCRIPTION
**Pull request by akshansh2000**
Added Pi Approximation using Monte Carlo method
Closes #54

### CODE REVIEW

File: lib/src/data/simulations.dart [hunk 1]
```diff
@@ -86,6 +88,15 @@ class Simulations with ChangeNotifier {
         infoLink: 'https://en.wikipedia.org/wiki/Fourier_series',
         fav: _favorites[5],
       ),
+      SimulationCard(
+        id: 6,
+        simulationName: 'Pi Approximation',
```
Comment (yashlamba) [inline lib/src/data/simulations.dart]: Simulation name is very generic, would be better if you could change it to something specific

**Review by yashlamba — CHANGES_REQUESTED** [review]
Rest looks good, but it never stabilizes over 2 digits even when I ran it for more that 3-4 mins, is it okay?

**Review by yashlamba — APPROVED** [review]

**Comment by yashlamba** [comment]
Planning to make any more changes?

**Comment by akshansh2000** [comment]
> Planning to make any more changes?

Nope. Anything else required?

**Comment by yashlamba** [comment]
> > Planning to make any more changes?
> 
> Nope. Anything else required?

nope

**Pull request merged**